website/content/api-docs/system/managed-keys.mdx

```diff
@@ -0,0 +1,194 @@
+that the key is supposed to use such as pkcs11. The :name argument is unique name within all managed key types within
+the namespace.
+
+| Method | Path                            |
+|:-------|:--------------------------------|
+| `POST` | `/sys/managed-keys/:type/:name` |
+
+### Sample Request
+
+```shell-session
+$ curl \
+    --request POST \
+    --header "X-Vault-Token: ..." \
+    http://127.0.0.1:8200/v1/sys/managed-keys/pkcs11/hsm-key1
+```
+
+### Generic Parameters
+- `name` `(string: <required>)` - A unique lowercase name that serves as identifying the key. Must be
+  unique throughout the namespace.
+
+- `type` `(string: <required>)` – The backend type that will be leveraged for the managed key.
+  Supported options are `pkcs11`.
+
+- `allow_generate_key` `(string: "false")` - If no existing key can be found in the referenced backend, instructs
+  Vault to generate a key within the backend.
+
+- `allow_replace_key` `(string: "false")` - Controls the ability for Vault to replace through generation or importing
+  a key into the configured backend even if a key is present, if set to false those operations are forbidden
+  if a key exists.
+
+- `allow_store_key` `(string: "false")` - Controls the ability for Vault to generate or import a key within the
+  configured backend, if "false" those operations will be forbidden.
+
+- `any_mount` `(string: "false")` - Allow usage from any mount point within the namespace if "true". If "false"
+  specific mount points will need to be tuned setting `allowed_managed_keys` with the name value of the key
+  granting usage.
+
+#### PKCS#11 backend Parameters
+- `type` `(string: "pkcs11")` - To select a PKCS#11 backend, the type parameter must be set to `pkcs11`
+
+- `library` `(string: <required>)` - The name of the `kms_library` stanza to use from Vault's config to
+   lookup the local library path. See [kms_library stanza](/docs/configuration/kms-library) for further details.
+
+- `key_label` `(string: <required>)`: The label of the key to use. If the key
+  does not exist and generation is enabled, this is the label that will be given
+  to the generated key.
+
+- `mechanism` `(string: <required>)` - The encryption/decryption mechanism to use,
+  specified as a hexadecimal (prefixed by 0x) string. The following are supported mechanisms
+    - `0x1041` `CKM_ECDSA`
+    - `0x000D` `CKM_RSA_PKCS_PSS`
+    - `0x0009` `CKM_RSA_PKCS_OAEP`
+    - `0x0001` `CKM_RSA_PKCS`
+
+- `pin` `(string: <required>)`: The PIN for login.
+
+- `slot` `(string: <slot or token label required>)`: The slot number to use,
+  specified as a string (e.g. `"2305843009213693953"`).
+
+- `token_label` `(string: <slot or token label required>)`: The slot token label to
+  use.
+
+- `curve` `(string: "")` - supplies the curve value when using the `CKM_ECDSA`
+  mechanism. This key is required if  The supported values are
```

Comment:
The sentence is cut off after "This key is required if".

Reply:
those darn squirrels... Fixed.